README.rst

```diff
@@ -11,22 +11,47 @@ lcls_naming_tool
 
 `Documentation <https://pcdshub.github.io/lcls_naming_tool/>`_
 
-A tool that checks the form and content of names with respect to the LCLS naming convention.
+A tool that checks the form and content of PV and device names with respect to the LCLS naming convention.
+
+Instructions
+------------
+
+Use at the command line:
+
+1. Activate the custom virtual environment with ``source /cds/group/pcds/pyps/conda/venvs/lcls_naming_tool/bin/activate``
+
+2. To check if a PV or device name is valid pipe in the name like so ``$ echo "MR2K4:KBO:PIP:01:PUMPSIZE" | ./lcls_naming_tool.py``
+
+3. To check if a list of names is valid pipe in the file name like so ``$ cat pvlist.txt | ./lcls_naming_tool.py`` (names should be separated by newline characters.)
+
+4. To view the current version add ``-v`` or ``--version`` like so ``$ echo "MR2K4:KBO:PIP:01:PUMPSIZE" | ./lcls_naming_tool.py -v``
+
+To run Flask and Gunicorn web server:
+
+``ssh psca@psctlws01``
+
+``cd /u1/psca/test/apps/nmsvc``
+
+``git pull``
+
+``./startup.sh``
+
+Website is hosted at https://psdm.slac.stanford.edu/lcls_naming_tool/
+
 
 Requirements
 ------------
 
+* PCDS conda
```

Comment:
Is this true? Are there specific packages unlisted here that are needed but are contained inside pcds conda?

Reply:
Hmm I think it was just flask and gunicorn. Removed this.